css-forms-1/Overview.bs

```diff
@@ -677,14 +677,18 @@ input:is([type=checkbox]:not([switch]), [type=radio]) {
     display: inline-flex;
     align-items: center;
     justify-content: center;
+    content: '';
+}
+
+input[type=radio] {
+		border-radius: 100%;
```

Comment:
Potentially want some sort of padding on this too?

Reply:
I intentionally didn't put any so it's not annoying to override, but perhaps it might be a good idea. I haven't thought too much about it

Reply:
Yeah I'll leave it without for now. We can revisit once the styles are more fleshed out as an overall quality question.